dnplab/dnpImport/prospa.py

```diff
@@ -239,3 +228,62 @@ def import_csv(path, return_raw=False, is_complex=True):
         return x, data
     else:
         return raw
+
+
+def prospa_coords(attrs, data_shape):
+    """Generate coords from prospa acquisition parameters"""
```

Comment:
Need docstring for `attrs` and `data_shape`. 
Meanwhile, it is assumed that `attrs["nrPnts"]` and `attrs["dwellTime"]` are `integer` and `float` respectively. If somehow one of them becomes, say `string`, then it will raise some confusing Exceptions. 

Ideally, there needs to be a safeguard asserting that. And better write a test for that.